src/components/regions/Regions.js

```diff
@@ -0,0 +1,123 @@
+import React, { useEffect, useState } from 'react';
+import { shallowEqual, useSelector, useDispatch } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import Pagination from 'react-bootstrap/Pagination';
+import Region from './Region';
+import Header from '../Header';
+import Loading from '../Loading';
+import { fetchCountriesAction } from '../../redux/countries/countries';
+
+const Regiones = () => {
+  const { id } = useParams();
+  const dispatch = useDispatch();
+  const { countries, loaded } = useSelector((state) => state.countries, shallowEqual);
+  const [regions, setRegions] = useState(null);
+  const [activePage, setActivePage] = useState(1);
+
+  useEffect(() => {
+    dispatch(fetchCountriesAction());
+    if (loaded) {
+      const countryObj = countries.filter((obj) => obj.id === id);
+      // Non camelCase are needed here since they're used by the Narrativa API.
+      /* eslint-disable camelcase */
+      const regionsData = countryObj[0].regions.map(({
+        id, name, today_new_confirmed, today_new_deaths, date,
+      }) => ({
+        date,
+        id,
+        name,
+        todayNewConfirmed: today_new_confirmed || 0,
+        todayNewDeaths: today_new_deaths || 0,
```

Comment:
You can alleivate the eslint camelCase error by doing this; renaming the variables as you destructure
```suggestion
      const regionsData = countryObj[0].regions.map(({
        id, name, today_new_confirmed: todayNewConfirmed, today_new_deaths: todayNewDeaths, date,
      }) => ({
        date,
        id,
        name,
        todayNewConfirmed: todayNewConfirmed || 0,
        todayNewDeaths: todayNewDeaths || 0,
```